docs/customize.md

```diff
@@ -9,7 +9,7 @@ The original mechanism for modifying kubernetes jobs using the Job Launcher is t
 [`job-json-overrides`](https://docs.posit.co/ide/server-pro/job_launcher/job_launcher.html).
 
 [There is a support article discussing the topic and examples in a legacy context 
-here](https://support.rstudio.com/hc/en-us/articles/360051652094-Using-Job-Json-Overrides-with-RStudio-Server-Pro-and-Kubernetes)
+here](https://docs.posit.co/ide/server-pro/job_launcher/kubernetes_plugin.html#kube-json)
```

Comment:
Should probably update the link text too, since that is not a support article any longer.